client/src/components/Upload/RulesInput.vue

```diff
@@ -22,18 +22,16 @@
                 </div>
             </div>
             <div id="upload-rule-dataset-option" class="upload-rule-option" v-if="selectionType == 'dataset'">
-                <div class="upload-rule-option-title">{{ l("Select dataset to load") }}</div>
-                <div class="dataset-selector">
-                    <select2
-                        v-model="selectedDatasetId"
-                        container-class="upload-footer-selection"
-                        placeholder="Select Dataset"
-                    >
-                        <option></option>
-                        <option v-for="dataset of datasets" :key="dataset.id" :value="dataset.id">
-                            {{ dataset.hid }}: {{ dataset.name }}</option
-                        >
-                    </select2>
+                <div class="upload-rule-option-title">
+                    History dataset
+                </div>
+                <div>
+                    <b-link @click="onSelectDataset" v-if="selectedDatasetName == null">
+                        {{ l("Select") }}
```

Comment:
Should we actually always do that? (maybe enforce even, it's the first time I see localization call in galaxy)

Reply:
We should always do that I think, I've gotten bad at it.